Source/Message/MessageClient.swift

```diff
@@ -37,7 +49,7 @@ open class MessageClient: CompletionHandlerType<Message> {
     /// - parameter queue: The queue on which the completion handler is dispatched.
     /// - parameter completionHandler: A closure to be executed once the request has finished.
     /// - returns: Void
-    open func list(roomId: String, before: String? = nil, beforeMessage: String? = nil, max: Int? = nil, queue: DispatchQueue? = nil, completionHandler: @escaping ArrayHandler) {
+    public func list(roomId: String, before: String? = nil, beforeMessage: String? = nil, max: Int? = nil, queue: DispatchQueue? = nil, completionHandler: @escaping ArrayHandler) {
```

Comment:
There are a few reasons. Originally when the SDK was written in Swift 2, there WAS no notion of public vs. open, and when we upgraded to 3 we did a somewhat mechanical transition. Things that were public in Swift 2 are logically open in Swift 3. However, the semantics are such that we probably don't actually need them to be open in most cases. In many of the cases, it's meaningless, as we don't expose inits for subclassing. In all cases I've seen, the SDK classes are really designed or intended for subclassing.

In some situations, making classes open would have required us to make public some classes we didn't want to expose through the init. I'm pretty sure we're okay moving most things to public rather than open though. Let me know if I'm wrong of course!